index.html

```diff
@@ -686,6 +686,51 @@ <h4>Redirected and Proxied External Content</h4>
           </p>
         </section>
       </section>
+
+      <section id="state-tokens">
+        <h3>State Tokens</h3>
+        <blockquote class="informative">
+          Non-normative note: State tokens are similar to ETags, providing a mechanism for clients to detect changes
+          in resource state without having to compare the entire entity body, and to perform conditional updates.
+          Like ETags, the principle behind state tokens is that implementations have the best information on
+          resource state, and can choose the most accurate and efficient mechanism for clients to detect changes.
+          State tokens differ from ETags in that there is no expectation that the state token must change if
+          the resource is not byte-for-byte identical, when the underlying state of the resource has not changed.
+          For example, changes in the order of RDF triples, blank node identifiers, and other serialization
+          variations would not require the <code>X-State-Token</code> value to change.
+        </blockquote>
+        <blockquote class="informative">
+          Non-normative note: State tokens can be a more reliable mechanism for detecting changes in resource state
+          than ETags when an implementation cannot provide strong ETags for resources. State tokens can be a more
```

Comment:
suggest deleting "for resources" in _"... cannot provide strong ETags ~for resources~."_ (seems easier the mentioning entity bodies again)